extension/maxminddb.c

```diff
@@ -166,6 +171,106 @@ static PyObject *Reader_get(PyObject *self, PyObject *args)
     return py_obj;
 }
 
+static int ip_converter(PyObject *obj, struct sockaddr_storage *ip_address)
+{
+#if PY_MAJOR_VERSION >= 3
+    if (PyUnicode_Check(obj)) {
+        Py_ssize_t len;
+        const char *ipstr = PyUnicode_AsUTF8AndSize(obj, &len);
+# else
+    if (PyUnicode_Check(obj) || PyString_Check(obj)) {
+        // Although this should work on Python 3, we will hopefully delete
+        // this soon and the Python 3 version is cleaner.
+        const char *ipstr = PyString_AsString(obj);
+        Py_ssize_t len = strlen(ipstr);
```

Comment:
Would `PyString_Size` or something be more correct? Just thinking whether it is talking about bytes or not, and whether there could be internal NULs. But I don't know!

Reply:
I thought I tried it and it didn't work for some reason, but it seems to work now. I was trying to fix Python 2 support before my self-imposed deadline on Friday though so I was a bit rushed.